R/utils.R

```diff
@@ -129,31 +129,39 @@ render_dependencies <- function(dependencies, local = TRUE) {
     } else {
       "file"
     }
+    
+    # According to Dash convention, label react and react-dom as originating
+    # in dash_renderer package, even though all three are currently served
+    # up from the DashR package
+    if (dep$name %in% c("react", "react-dom")) {
+      dep$name <- "dash_renderer"
+    }
+
+    # The following lines inject _dash-component-suites into the src tags,
+    # as this is the current Dash convention. The dependency paths cannot
+    # be set solely at component library generation time, since hosted
+    # applications should have the app name injected as well.
+    #
+    # This is essentially analogous to this codeblock on the Python side:
+    # https://github.com/plotly/dash/blob/1249ffbd051bfb5fdbe439612cbec7fa8fff5ab5/dash/dash.py#L207
+    if (is.null(prefix)) {
```

Comment:
we should be able to remove this `is.null` check - by the time this function is called, we've already coerced `requests_pathname_prefix` from null to `/` in `resolve_prefix`

Reply:
This definitely looks like an oversight on my part. Will fix, thanks for letting me know.

Reply:
fixed in 0569677c435c928878577db110b4036c9b0b12a8